### DISCUSSION TITLE
Zwave 700 support

### DISCUSSION THREAD

**Discussion by DrSamjames** [zwave-js/zwave-js#2134] [Q&A]
Hi

hopefully this a new question, but what level of support is there for Z-wave 700 at the moment, specifically does anything work with the Z-Pi 7 by aeotec:

https://aeotec.com/z-wave-home-automation/development-kit-pcb.html

I have searched and I can find posts from a year ago saying that it would be about a year before any support was likely to be available from any of the Zwave projects. I also see in the change log "Added basic support for 700-series controllers"?

Thanks

Sam

**Comment by AlCalzone** [comment]
If you don't need any special functionality, you can use the 700 series sticks right now. In fact I am using one for development.

**Reply by DrSamjames** [reply]
Great thanks. I'm doing work on a home automation project not specifically developing zwave. So mainly for the moment just having it run normal gen 5 devices is all that is needed.
